app/src/main/java/org/mozilla/fenix/components/metrics/GleanMetricsService.kt

```diff
@@ -758,6 +758,18 @@ class GleanMetricsService(
                 topSitesCount.add(topSitesSize)
             }
 
+            val desktopBookmarksSize = context.settings().desktopBookmarksSize
+            hasDesktopBookmarks.set(desktopBookmarksSize > 0)
+            if (desktopBookmarksSize > 0) {
```

Comment:
Why not just use the if (hasDesktopBookmarks) here?

Reply:
hasDesktopBookmarks is the metric code. We never set a variable for desktopoBookmarksSize > 0